pkg/driver/node.go

```diff
@@ -208,9 +208,9 @@ func (d *nodeService) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 		// Special case for xfs default format and mout fails due to fromat
 		// mkfs -t xfs -m reflink=0 /dev/xvdh
```

Comment:
Is this special case still occurs with this new version?

Reply:
After a second test, this special case is not necessary